fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/api/registry/FlammableBlockRegistry.java

```diff
@@ -21,15 +21,19 @@
 import net.minecraft.tag.Tag;
 
 import net.fabricmc.fabric.api.util.Block2ObjectMap;
-import net.fabricmc.fabric.impl.content.registry.FlammableBlockRegistryImpl;
+import net.fabricmc.fabric.impl.content.registry.LegacyFlammableBlockRegistryImpl;
 
+/**
+ * @deprecated replaced by {@link net.fabricmc.fabric.api.content.registry.v1.FlammableBlockRegistry}
+ */
+@Deprecated
 public interface FlammableBlockRegistry extends Block2ObjectMap<FlammableBlockRegistry.Entry> {
 	static FlammableBlockRegistry getDefaultInstance() {
 		return getInstance(Blocks.FIRE);
 	}
 
 	static FlammableBlockRegistry getInstance(Block block) {
-		return FlammableBlockRegistryImpl.getInstance(block);
+		return LegacyFlammableBlockRegistryImpl.getInstance(block);
```

Comment:
Why rename this?

Reply:
It conflicts with the new one when both old and new modules are present

Reply:
shoudlnt do, they are in diffrent packages. We havent done this for any other module.

Reply:
They are in the same package though. Only api is split by `v0` and `v1`. Impl and mixin use the same packages.

Reply:
humm, moving them to a new package might be better? im not too fond of the Legacy prefix.

Reply:
I think that would violate the checkstyle? not sure though. either way, what does it matter as they're impl that will be removed eventually anyway. Really just needs to not conflict